src/vc.rs

```diff
@@ -1904,6 +1923,67 @@ impl Presentation {
         }
         results
     }
+
+    /// Like [get_verification_methods_for_purpose] but including VMs of DIDs authorized to
+    /// act as holder via [Presentation::holder_binding].
+    async fn get_verification_methods_for_purpose_bindable(
+        &self,
+        resolver: &dyn DIDResolver,
+        proof_purpose: ProofPurpose,
+    ) -> Result<Vec<String>, Error> {
+        let authorized_holders = self.get_authorized_holders().await?;
+        let vmms = crate::did_resolve::get_verification_methods_for_all(
+            authorized_holders
+                .iter()
+                .map(|x| x.as_str())
+                .collect::<Vec<&str>>()
+                .as_ref(),
+            proof_purpose.clone(),
+            resolver,
+        )
+        .await?;
+        Ok(vmms.into_keys().collect())
+    }
+
+    pub(crate) async fn get_authorized_holders(&self) -> Result<Vec<String>, Error> {
+        let mut holders = match (self.holder.as_ref(), self.holder_binding.as_ref()) {
+            (Some(_), Some(_)) | (None, None) => vec![],
+            (Some(h), None) => vec![h.to_string()],
+            (None, Some(_)) => return Err(Error::MissingHolder),
+        };
+        for holder_binding in self.holder_binding.iter().flatten() {
+            match &holder_binding {
+                HolderBinding::ExampleHolderBinding2022 { to, from } => {
```

Comment:
I'm not sure what we should do about this, maybe remove it altogether? I suppose the main question is what kind of proof is it supposed to be?

Reply:
I've hidden all usage of it behind `#[cfg(test)]`